module-system/module-implementations/sov-accounts/src/lib.rs

```diff
@@ -12,17 +12,18 @@ pub use query::*;
 mod tests;
 
 pub use call::{CallMessage, UPDATE_ACCOUNT_MSG};
-use sov_modules_api::{Context, Error, ModuleInfo, PublicKeyHex, WorkingSet};
+use sov_modules_api::{Context, Error, ModuleInfo, WorkingSet};
 
 /// Initial configuration for sov-accounts module.
 #[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
-pub struct AccountConfig {
+#[serde(bound = "C::PublicKey: serde::Serialize + serde::de::DeserializeOwned")]
+pub struct AccountConfig<C: Context> {
```

Comment:
What do you think about making this generic over the public key type itself? It might remove some boilerplate code.

Reply:
I tried and eded up with:
```rust
#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
#[serde(bound = "PK: serde::Serialize + serde::de::DeserializeOwned")]
pub struct AccountConfig<PK: PublicKey> {
    /// Public keys to initialize the rollup.
    pub pub_keys: Vec<PK>,
}
```
So `#[serde(bound = "PK: serde::Serialize + serde::de::DeserializeOwned")]` is still needed.